package com.paymentsystem.authservice.kafka.event;

import com.paymentsystem.authservice.domain.enums.Currency;
import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class UserRegisteredEvent {
    private UUID id;
    private String name;
    private String email;
    private Currency currency;
}
